index.bs

```diff
@@ -631,20 +636,24 @@ To <dfn>abort the request</dfn> |request|:
 
 1. [=Assert=]: these steps are running on the [=lock task queue=].
 1. Let |origin| be |request|'s [=lock request/origin=].
-1. [=list/Remove=] |request| from |origin|'s [=lock request queue=].
-1. [=Process the lock request queue=] for |origin|.
+1. Let |name| be |lock|'s associated [=resource name=].
+1. Let |queue set| be |origin|'s [=lock request queue set=].
+1. Let |queue| be |name|'s associated [=lock request queue=] in |queue set|.
+1. [=list/Remove=] |request| from |queue|.
+1. [=Process the lock request queue=] |queue|.
```

Comment:
need to pass in |origin| as an argument.

Reply:
Looks like the origin is exposed through |request|. Is the suggestion to pass it in for clarity?

Reply:
As currently written, the "process the lock request queue" algorithm takes |name| and |origin| as arguments, so you need to "pass those in" or restructure the algorithm's "method signature". 

I agree that now it doesn't need |origin| passed in, so you could remove that from the "method signature", i.e. remove the "in |origin|" phrase.

Reply:
Ah, sorry, I thought you meant that the 'abort the request' algorithm needed to take |origin| as an argument. I see what you mean now.